package.json

```diff
@@ -16,6 +16,7 @@
   "author": "Apache Software Foundation",
   "license": "Apache-2.0",
   "dependencies": {
+    "@cordova/eslint-config": "file:../cordova-eslint/cordova-eslint-config-3.0.1-dev.tgz",
```

Comment:
Is this change intended?

Reply:
Nope definitely not. I must have left my environment is a dirty state the last time I touched this repo :sweat_smile: 

At least someone checks the work despite how trivial this is ;)